package.json

```diff
@@ -7,11 +7,10 @@
         "dev": "next dev",
         "build": "next build",
         "start": "next start",
-        "lint": "next lint && prettier --check .",
-        "lint:fix": "next lint --fix && prettier --write .",
```

Comment:
how come no more next lint? just curious

Reply:
`next lint` gives an error when those frontend files are removed (`pages`) and (`apps`)